packages/userstorage/src/browser/user-storage-service-filesystem.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { DisposableCollection, ILogger, Emitter, Event } from '@theia/core/lib/common';
+import { UserStorageChangeEvent, UserStorageService, THEIA_USER_STORAGE_FOLDER } from './user-storage-service';
```

Comment:
`THEIA_USER_STORAGE_FOLDER` is implementation specific and should be defined unexposed in this file.

Reply:
Oh right, it doesn't belong in the interface file that's what you mean right? Moving it here with the implementation would make sense in that case

Reply:
Yes